src/main/webpack/scss.js

```diff
@@ -0,0 +1,26 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const miniCss = new MiniCssExtractPlugin({
+  // Options similar to the same options in webpackOptions.output
+  // both options are optional
+  filename: '[name].css',
+  chunkFilename: '[id].css',
```

Comment:
Follow up PR for hashing? Are cache control headers being set currently?

Reply:
yes, will do in a separate PR